# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/jimmy/mplayer/vc-2.6-wkround.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./libvo/vo_svga.c.orig	2004-06-13 17:33:53.944331344 +0300
+++ ./libvo/vo_svga.c	2004-06-13 17:33:47.920247144 +0300
@@ -395,6 +395,8 @@
   return VO_NOTIMPL;
 }
 
+extern char MSGTR_Paused_r[24];
+
 //
 // This function is called to init the video driver for specific mode
 //
@@ -449,6 +451,7 @@
     uninit();
     return 1; // error
   }
+  *MSGTR_Paused_r = '\r';
   /* set 332 palette for 8 bpp */
   if(mode_bpp==8){
     int i;
--- ./mplayer.c.orig	2004-04-27 00:15:13.000000000 +0300
+++ ./mplayer.c	2004-06-13 17:30:31.619089456 +0300
@@ -707,6 +707,8 @@
 #include <SDL.h>
 #endif
 
+char MSGTR_Paused_r[24] = MSGTR_Paused;
+
 
 int main(int argc,char* argv[]){
 
@@ -2409,7 +2411,7 @@
   if(osd_function==OSD_PAUSE){
     mp_cmd_t* cmd;
       if(!quiet) {
-	mp_msg(MSGT_CPLAYER,MSGL_STATUS,MSGTR_Paused);
+	mp_msg(MSGT_CPLAYER,MSGL_STATUS,MSGTR_Paused_r);
 	fflush(stdout);
       }
 #ifdef HAVE_NEW_GUI
